.github/workflows/maven-deploy.yml

```diff
@@ -33,9 +34,9 @@ jobs:
       - name: Check Maven and Java versions
         run: mvn -v
 
-      - run: mvn clean deploy --batch-mode --activate-profiles release -Dgpg.signer=bc
+      - run: mvn -X -Dorg.slf4j.simpleLogger.log.org.apache.http.headers=debug -Dorg.slf4j.simpleLogger.log.org.apache.maven.wagon.providers.http.httpclient=debug clean deploy --batch-mode --activate-profiles release -Dgpg.signer=bc
```

Comment:
Was the debug code only for earlier exploration purposes?

Reply:
Fixed by commit 62cde261